src/app/dialog-service/dialog.service.ts

```diff
@@ -5,31 +5,42 @@ import { ModalComponent } from './modal/modal/modal.component';
 import dialogData from '../../assets/json/dialogData.json';
 import { Router } from '@angular/router';
 
-
 @Injectable({
   providedIn: 'root',
 })
 export class DialogService {
   adminform: any;
-  
-  constructor(
-    private modalService:NgbModal,
-    private route:Router
-  ) {}
 
-  dialogData = {...dialogData};
+  constructor(private modalService: NgbModal, private route: Router) {}
+
+  dialogData = { ...dialogData };
 
-  openDialog(label:string, yesButtonLable:string, NoButtonLable:string):Promise<boolean>{
+  openDialog(
+    label: string,
+    yesButtonLable: string,
+    NoButtonLable: string
+  ): Promise<boolean> {
     return new Promise((resolve, reject) => {
       const modalRef = this.modalService.open(ModalComponent);
       modalRef.componentInstance.label = label;
       modalRef.componentInstance.yesButtonLable = yesButtonLable;
       modalRef.componentInstance.NoButtonLable = NoButtonLable;
-      
-      modalRef.result.then((data) => {
-        resolve(<boolean> data)
-      }); 
-    })
+
+      modalRef.result
+        .then((data) => {
```

Comment:
Could you please change data to answer ?

Reply:
Yes i have changed it